### PR TITLE
Reduce image quality to optimise download data requirements

### PR DESCRIPTION
**Pull request by alloy**
Following https://github.com/artsy/force/pull/2226 I’ve reduced the quality here to 80 as well and seeing about a ~19.5% reduction in downloaded data.

#skip_new_tests

### CODE REVIEW

File: Example/Emission/AppSetup.m [hunk 1]
```diff
@@ -30,19 +30,17 @@ - (instancetype)initWithAmbientContext
...
 #endif
 
-
+    // Comment these out to set yourself up as though you were running the beta
+    usePRBuild = [defaults boolForKey:ARUsePREmissionDefault];
     useMaster = useMaster || isSimulator;
-
     useRNP = isSimulator || [defaults boolForKey:ARForceUseRNPDefault];
 
     if (usePRBuild) {
```
Comment (orta) [inline Example/Emission/AppSetup.m]: ^

Comment (alloy) [reply Example/Emission/AppSetup.m]: ?

Comment (orta) [reply Example/Emission/AppSetup.m]: oh yeah, it just moves the logic down - cooooool, ignore me

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/OpaqueImageView-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Example/Emission/AppSetup.m (hunk 1 above)
Comment (alloy) [reply Example/Emission/AppSetup.m]: Would have been so epic if we could’ve finished this discussion using only single characters!

Comment (orta) [reply Example/Emission/AppSetup.m]: Maybe a headshake? Or a nod? Not quite sure how I could have expressed I was wrong, I agree in a single char

Comment (mennenia) [reply Example/Emission/AppSetup.m]: !

File: Example/Emission.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -565,7 +565,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [ \"$CONFIGURATION\" = \"Release\" ]\nthen\ncd .. && npm run bundle\nfi";
+			shellScript = "if [ \"$CONFIGURATION\" = \"Release\" ]\nthen\nexport NVM_DIR=\"$HOME/.nvm\"\n[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"\ncd .. && nvm use && npm run bundle\nfi";
```
Comment (mennenia) [inline Example/Emission.xcodeproj/project.pbxproj]: this is to load local bundled JS in a release build, right?

File: src/lib/Components/OpaqueImageView.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {
 import colors from "lib/data/colors"
 
 const GeminiHost = "d7hftxdivxxvm.cloudfront.net"
-const ImageQuality = 85
+const ImageQuality = 80
```
Comment (mennenia) [inline src/lib/Components/OpaqueImageView.tsx]: 🎉

File: Example/Emission.xcodeproj/project.pbxproj (hunk 2 above)
Comment (alloy) [reply Example/Emission.xcodeproj/project.pbxproj]: Yup

**Pull request merged**